install.sh

```diff
@@ -29,25 +29,30 @@ main()
     # Never empty, dirname returns "." instead (opengroup.org)
     local path; path=$(dirname "$1")
     local ver; ver=$(basename "$1")
-    local sdir sloc optversuffix
+    local sdir optversuffix
 
     [ -z "$ver" ] || optversuffix="-$ver"
 
-    for sdir in sof sof-tplg; do
-        sloc="$path/${sdir}${optversuffix}"
-        test -d "$sloc" ||
-            die "%s not found\n" "$sloc"
-    done
-
     # Do this first so we can fail immediately and not leave a
     # half-install behind
     if [ -n "$optversuffix" ]; then
     set -x
-    for sdir in sof sof-tplg; do
-        ln -sT "$sdir-$ver" "${FW_DEST}/$sdir" || {
-            set +x
-            die '%s already installed? (Re)move it first.\n' "${FW_DEST}/$sdir"
-        }
+
+    if test -e "$path/sof${optversuffix}" -a -e "$path/sof-tplg${optversuffix}" ; then
```

Comment:
I kept the original and now wrong indentation to preserve `git blame` but now that all the code is changed there's no reason  not to indent correctly now.

Reply:
@marc-hb I was wondering why you kept the intendation. Now fixed.